<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>obnova</class>
 <widget class="QDialog" name="obnova">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Магазины</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить запись о магазине с ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Изменить поле:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>  Новое значение поля:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNewMagaz">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ObnovaIDmagaz">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ObnovaPoleMagaz">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushObMagaz">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить запись</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>    Товары    </string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить запись о товаре с ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить поле:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Новое значение поле:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ObnovaIDtovar">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ObnovaPoleTovar">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNewTovar">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushObTovar">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить запись</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>     Заказы     </string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить запись о заказе с ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Изменить поле:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>  Новое значение поле:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ObnovaIDzakaz">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ObnovaPolezakaz">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNewZakaz">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushObZakaz">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить запись</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>      Доставки    </string>
    </attribute>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить запись о доставки с ID:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ObnovaIDdost">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ObnovaPoleDost">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить поле:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Новое значение поле:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNewDost">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushObDost">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить запись</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
